www/js/pages/maltainvest-account-opening/maltainvest-account-opening.controller.js

```diff
@@ -46,7 +46,8 @@
         vm.hasResidence = false;
         vm.hasPOB = false;
         const loginid = accountService.getDefault().id;
-        const isVirtual = clientService.isAccountOfType('virtual', loginid);
+        const landingCompany = localStorage.getItem('landingCompany');
+        const isVirtual = clientService.isAccountOfType('virtual', loginid, landingCompany) || landingCompany === 'virtual';
```

Comment:
@nazaninreihani 
You're passing 3 arguments to `clientService.isAccountOfType` function but the main function in `client.service.js` just accepts two arguments. 

Do you have any plans for further changes in the function's body?